packages/botonic-plugin-knowledge-bases/src/index.ts

```diff
@@ -20,24 +20,27 @@ export default class BotonicPluginKnowledgeBases implements Plugin {
 
   async getInference(
     session: Session,
-    userInput: string,
-    sources: string[]
+    sources: string[],
+    instructions: string,
+    messageId: string,
+    memoryLength: number
   ): Promise<KnowledgeBaseResponse> {
     const authToken = isProd ? session._access_token : this.authToken
 
```

Comment:
I would add a safe check here (if process.env === 'development') return 
with a TODO for local version

Reply:
I don't understand you, isProd is checking if `process.env === 'production'` which is the case when we want to use the token in session